dashboard/app/views/levels/editors/fields/_default_song.html.haml

```diff
@@ -22,4 +22,4 @@
 
 %p Previously chosen default song (may not match actual default if song selection was limited):
 
-= songMap[@level.default_song][0]
+= (songMap[@level.default_song]||[])[0]
```

Comment:
This is ugly syntax. Does ruby have a slicker construct? Something like a nullish coalescing operator?